test/unit/presenters/publishing_api_presenters/links_presenter_test.rb

```diff
@@ -1,18 +1,7 @@
 require 'test_helper'
 
 class PublishingApiPresenters::LinksPresenterTest < ActionView::TestCase
-  ALL_LINK_TYPES = [
-    :document_collections,
-    :lead_organisations,
-    :policy_areas,
-    :related_policies,
-    :statistical_data_set_documents,
-    :supporting_organisations,
-    :topics,
-    :world_locations,
-    :worldwide_organisations,
-    :worldwide_priorities,
-  ]
+  ALL_LINK_TYPES = PublishingApiPresenters::LinksPresenter::LINK_NAMES_TO_METHODS_MAP.keys
```

Comment:
Not especially. If anything there's probably a test missing for an item's organisation IDs. But the `ALL_LINK_TYPES` constant is simply set for convenience - with the change above it doesn't need to be updated for every addition to the `LINK_NAMES...` map, and its value now more accurately reflects its name.